File: Watermelon.java
package PCPH;

import java.util.Scanner;

public class Watermelon {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        int n= sc.nextInt();
        String var = (n>2)?((n%2==0)? "YES":"NO"):"NO";
        System.out.println(var);
    }
}

File: Watermelon.c
#include<stdio.h>
int main(){
    int a;
    scanf("%d",&a);
    if(a>2){
        if(a%2==0){
            printf("YES");
        }
        else{
            printf("NO");
        }
    }
    else{
        printf("NO");
    }
    
    return 0;
}
